### PR TITLE
Aha/update to v3

### PR DESCRIPTION
**Pull request by Alexander89**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -357,7 +352,7 @@ export const httpConnector = (config: HttpConnectorConfig): expressWs.Applicatio
         }
       }
       const { tags, payload } = data
-      console.log(tags, payload)
+      // console.log(tags, payload)
```
Comment (wwerner) [inline src/index.ts]: Would remove instead of commenting out.

File: src/index.ts [hunk 2]
```diff
@@ -237,7 +238,7 @@ export type HttpConnectorConfig = {
 /**
  * @ignore
  */
-const bodyLimit = '20mb'
+export const bodyLimit = '20mb'
```
Comment (wwerner) [inline src/index.ts:241]: If I read it correctly, this is the _request_ body parser limit, right?
Why is it 20MB? I thought 4MB was the event size maximum.

File: add-license-text [hunk 3]
```diff
@@ -1,5 +1,5 @@
 
-Copyright 2020 Actyx AG
+Copyright 2021 Actyx AG
```
Comment (wwerner) [inline add-license-text:2]: Is there a specific reason for choosing the Apache license? Asking b/c I'd typically default to MIT. Saw Academy-Data is also under Apache.

**Comment by wwerner** [comment]
LGTM.
Love that you added tests and CI! <3

Just one question about the request body size.

**Review by wwerner — COMMENTED** [review]
LGTM.
Love that you added tests and CI! <3

Just one question about the request body size.

File: src/index.ts (hunk 2 above)
Comment (Alexander89) [reply src/index.ts:241]: it is not so easy. This is an Actyx best practice value. 
4MB compressed size. + header and other stuff In the emitter.

I think 20 MB is an upper bound with some wired background. :see_no_evil:

File: add-license-text (hunk 3 above)
Comment (Alexander89) [reply add-license-text:2]: We had this conclusion actyx internal after setting the Pond lic.

**Review by wwerner — APPROVED** [review]
LGTM

**Pull request merged**